sound/soc/sof/sof-priv.h

```diff
@@ -319,7 +328,7 @@ struct snd_sof_dev {
 
 	/* DSP firmware boot */
 	wait_queue_head_t boot_wait;
-	u32 boot_complete;
+	enum snd_sof_fw_state fw_state;
 	u32 first_boot;
 
```

Comment:
@ranj063 can we say that if state is BOOT_COMPLETE than first_boot = true? and then we can remove the first_boot variable?

Reply:
@dbaluta sorry I'm a bit confused about the first statement. first_boot is mainly used to avoid creating the debugfs entries during resume (both system and runtime resume). The state BOOT_COMPLETE would be reached every time the FW boots during a resume.

Reply:
@ranj063 thanks for the explanations, disregard my comment. :)

Reply:
@dbaluta the patch looks OK then?

Reply:
Yes, thanks! Just marked the PR as ok!